AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-edit/identity-edit.component.html

```diff
@@ -61,7 +61,7 @@ <h2 class="header-title">{{ header }}</h2>
                         </mat-checkbox>
                     </th>
                     <td mat-cell *matCellDef="let row; let i = index">
-                        <mat-checkbox (click)="$event.stopPropagation()"
+                        <mat-checkbox *ngIf="row.source === 'Individual'" (click)="$event.stopPropagation()"
```

Comment:
Wouldn't it make more sense to have a `deletable` or `canBeDeleted` property on the model? I don't think that it is a good idea to have such logic in the UI.